packages/flutter_tools/test/general.shard/device_test.dart

```diff
@@ -447,24 +411,22 @@ void main() {
       );
 
       expect(filtered.single, ephemeralOne);
-      verify(mockDeviceDiscovery.devices).called(1);
-      verifyNever(mockDeviceDiscovery.discoverDevices(timeout: anyNamed('timeout')));
+      expect(deviceDiscovery.devicesCalled, 1);
+      expect(deviceDiscovery.discoverDevicesCalled, 0);
     });
 
     testUsingContext('refreshes device cache with a timeout', () async {
```

Comment:
I believe the two `testUsingContext` in this file can be made `testWithoutContext`, and then you can remove the `context.dart` import.

Reply:
Done!